.github/copilot-instructions.md

```diff
@@ -153,3 +153,23 @@ When facing issues, direct users to:
 - [Support for Azure SDK for Java](https://github.com/Azure/azure-sdk-for-java/blob/main/SUPPORT.md)
 - [GitHub Issues](https://github.com/Azure/azure-sdk-for-java/issues/new/choose)
 - [Stack Overflow with azure-java-sdk tag](https://stackoverflow.com/questions/tagged/azure-java-sdk)
+
+## SDK release
+
+There are two tools to help with SDK releases:
+- Check SDK release readiness
+- Release SDK
+
+### Check SDK Release Readiness
+Run `CheckPackageReleaseReadiness` to verify if the package is ready for release. This tool checks:
+- API review status
+- Change log status
+- Package name approval(If package is new and releasing a preview version)
```

Comment:
```suggestion
- Package name approval (If package is new and releasing a preview version)
```

Reply:
Will this tool also check version_client.txt status required for Java release?

Reply:
Any additional validation per language requires changes in Engsys side to add a step in the CI.